README.md

```diff
@@ -174,9 +174,9 @@ hanging in the legacy UI and stack traces from rbac_server:
     rbac-server    |     response = await response
```

Comment:
Does this remove the need for the -d option in bin/start? If so, please remove references to it in the README. Also, whatever you can add to README/Wiki to capture this change == Good stuff.

Reply:
Updated